### PR TITLE
Adjusted event thumbnail size

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes MozillaFoundation/NSF-wireless-innovation-challenge#31

### CODE REVIEW

File: source/pug/partials/events.pug [hunk 1]
```diff
@@ -5,29 +5,29 @@
 
 .container
   include:markdown ../../markdown/events.md
-  .events
-    .event-container
+  .events.row
+    .event-container.col-md-3.col-sm-6.col-6
```
Comment (mmmavis) [inline source/pug/partials/events.pug:9]: https://v4-alpha.getbootstrap.com/layout/grid/#grid-options

Bootstrap is based on 12-column grid. So `col-md-3` means "make this element 3-column wide on medium size devices"

File: source/pug/partials/events.pug [hunk 2]
```diff
@@ -5,29 +5,29 @@
 
 .container
   include:markdown ../../markdown/events.md
-  .events
-    .event-container
+  .events.row
```
Comment (mmmavis) [inline source/pug/partials/events.pug:8]: Bootstrap `col-` elements need a `.row` element as their immediate parent.

**Comment by ScottDowne** [comment]
I think I'm OK with this, however, my pull request did have one small additional change I'm hoping we can add to this.

This doesn't grow the font size from 1rem to 1.25rem when it goes above `$bp-md`. This is for the address link text, so it doesn't line break and can still fit when on small displays.

**Comment by ScottDowne** [comment]
We can update font size if we still want to in another pr.

**Comment by ScottDowne** [comment]
Hm, there are two other small issues I want to fix around the font and text for this.

**Pull request merged**